src/target/source/interface_c.cc

```diff
@@ -103,6 +109,9 @@ class InterfaceCNode : public runtime::ModuleNode {
   }
 
  private:
+  constexpr static const char* _macro_pool_size_postfix = "_POOL_SIZE_BYTES";
+  constexpr static const char* _macro_pool_data_postfix = "_POOL_DATA";
```

Comment:
Can you verify a debug build works with this ? 

We have had problems before. See https://github.com/apache/tvm/pull/10262

Reply:
It works.